woocommerce/woocommerce-ios
# [Woo POS] M2: Payment success UI update

Closes: #13344


## Description

Making payment success UI update. Leaving animation of the success icon out of scope.

## Solution

1. Added `wooCommerceEmerald` palette so it could be accessed the same way as `wooCommercePurple`
2. Updated success view spacing, and style
3. Replaced success image with SF symbol
4. Passed `Order` to `PointOfSaleCardPresentPaymentSuccessMessageViewModel` to show the order total. Made it testable by injecting currency formatter
5. Updated new order button layout
6. Added animations to transition from payment processing to success state

## Testing information

1. Open Woo -> POS
2. Connect Reader
3. Add items
4. Check Out
5. Pay the order
6. Confirm transition fades from payment processing to success view
8. Confirm success view looks according to the designs TfaZ4LUkEwEGrxfnEFzvJj-fi-2530_13218
9. Confirm correct order total is shown
10. Confirm new order button starts a new order

## Screenshots


https://github.com/user-attachments/assets/3e053dc4-5a3e-403f-8587-a2c3790f8f76

---
